packages/cozy-mespapiers-lib/src/components/Papers/PaperGroup.jsx

```diff
@@ -9,13 +9,15 @@ import Typography from 'cozy-ui/transpiled/react/Typography'
 
 import { useMultiSelection } from '../Hooks/useMultiSelection'
 import CategoryItemByPaper from './CategoryItemByPaper'
+import CategoryItemByKonnector from './CategoryItemByKonnector'
 
-const PaperGroup = ({ papersByCategories }) => {
+const PaperGroup = ({ papersByCategories, konnectors }) => {
   const navigate = useNavigate()
   const { t } = useI18n()
   const { isMultiSelectionActive, setSelectedThemeLabel } = useMultiSelection()
 
   const hasPapers = papersByCategories.length > 0
+  const konnectorsWithAccounts = JSON.parse(JSON.stringify(konnectors))
```

Comment:
Je ne comprends pas le besoin de faire ça ?

Reply:
c'est pour dupliquer le tableau en créant une nouvelle référence (et donc sans muter le premier tableau), ce qui ne foncitonne pas avec `[...array]`

Reply:
Hum ok, les performance ici sont sans doute négligeables, mais il y a toujours un risque de perte d'informations avec cette approche 🤔 
Il existe la méthode native `structuredClone` ou un `cloneDeep` de lodash, qu'en dis tu ?

Reply:
@Merkur39 🙏 . `structuredClone` FTW! 

@JF-Cozy https://medium.com/@pmzubar/why-json-parse-json-stringify-is-a-bad-practice-to-clone-an-object-in-javascript-b28ac5e36521

Reply:
je voulais éviter lodash puisque c'est ce qu'on veut faire si j'ai bien compris.
Ok merci je regarde pour `structuredClone`

Reply:
j'ai tout simplifié, voir dernier commit